stripe/src/main/java/com/stripe/android/FingerprintDataStore.kt

```diff
@@ -22,7 +22,7 @@ internal interface FingerprintDataStore {
             )
         }
 
-        override fun get() = liveData<FingerprintData?>(coroutineDispatcher) {
+        override fun get() = flow<FingerprintData?> {
```

Comment:
Does this need to be an observable stream? It appears to emit only one value. If that's the case, perhaps this could be a `suspend` function returning an `async` coroutine builder which can be `await()`ed on.